docs/faq.md

```diff
@@ -1,5 +1,137 @@
+	- This increases the amount of in-flight EBS volume creations / attaches / modifications / deletions managed by the driver (which may increase your risk of region-level ['Mutating action' request throttling by the Amazon EC2 API](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html#throttling-limits) )
+	- **Note: If you are running multiple clusters within a single AWS account and region and risk hitting your [EC2 API Throttling Limits](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html#throttling-increase)**. See [Request a limit increase](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html#throttling-limits-rate-based) and the below [Fine-tuning the CSI Sidecar worker-threads parameter](faq.md) section
+3. Increasing the default CSI sidecar `kube-api-qps` (to 20) and `kube-api-burst` (to 100) for all sidecars ([#1834](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1834))
+	- Each sidecar can now send a burst of up to 100 [queries](https://kubernetes.io/docs/reference/using-api/api-concepts/#api-verbs) to the Kubernetes API Server before throttling itself. It will then allow up to 20 more requests per second until it stops bursting.   
+	- This keeps Kubernetes objects (`PersistentVolume`, `PersistentVolumeClaim`, and `VolumeAttachment`) more synchronized with the actual state of AWS resources, at the cost of increasing the load on the K8s API Server from `ebs-csi-controller` pods when many volume operations are happening at once. 
+4. Increasing the default CSI sidecar `timeout` values (from 15s to 60s) for all sidecars ([#1824](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1824))   
+  - E.g. the [external-attacher](https://github.com/kubernetes-csi/external-attacher?tab=readme-ov-file#csi-error-and-timeout-handling) will now give the driver up to 60s to report an attachment success/failure before retrying a `ControllerPublishVolume` call. Now the external-attacher won't prematurely timeout a `ControllerPublishVolume` call that would've taken 20s before returning a success response.  
+	- This decreases the amount of premature timeouts for CSI RPC calls, which reduces the amount of replay EC2 API requests made by and waited for by the driver (at the cost of a longer delay during a real driver timeout (e.g. network blip leads to lost `ControllerPublishVolume` response))
+## EC2 and K8s CSI Sidecar Throttling Overview
+
+Both the EC2 API and K8s CSI sidecars base their API throttling implementation off of the [token bucket algorithm](https://en.wikipedia.org/wiki/Token_bucket). Both the [Amazon EC2 API Reference](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html#throttling-limits-rate-based) and the [K8s documentation definitions of qps and burst](https://kubernetes.io/docs/reference/config-api/apiserver-eventratelimit.v1alpha1/#eventratelimit-admission-k8s-io-v1alpha1-Limit) describe their throttling systems.  
```

Comment:
I'm thinking this information may be out of scope for an FAQ doc. In spirit of minimalism and conciseness I would suggest removing it, but would be interested to see other opinions.

Reply:
+1, especially given it links the wrong k8s docs

Reply:
> especially given it links the wrong k8s docs

I disagree that these are 'wrong' docs. 

Those k8s docs are the only ones I could find that clearly explain what qps and burst mean in the context of Kubernetes, and therefore a better resource than docs like [kubelet | Kubernetes](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)

Reply:
> I'm thinking this information may be out of scope for an FAQ doc. In spirit of minimalism and conciseness I would suggest removing it, but would be interested to see other opinions.

I disagree. I think that this section, when not expanded, is small enough to be helpful context rather than out of scope. I think it is required reading for understanding how to tune the sidecar parameters and understand one of the root causes of scalability issues. 

Furthermore, I'd rather have too much info than too little, since anyone looking at an FAQ is in the process of diving deep, and can move onto the next heading if desired. 

However, if either @wmesard, @rdpsin, or Manu agree with @torredil, I can commit to taking this section out.

Reply:
Those docs are about the API Server rate limiting, the client-side throttling on the sidecars happens to work similarly by accident.

Reply:
What I mean is that those API Server rate limiting docs featured the best explanation of what `burst` and `qps` mean in the context of K8s (because the explanation on the sidecar's documentation had some ambiguity on the relationship between burst and qps)

But I see how linking the documentation for a wholly different part of k8s could also be misleading. Will take out.